Postgresql/legos/postgresql_check_unused_indexes/__init__.py

```diff
@@ -0,0 +1,6 @@
+#
```

Comment:
This file should be empty @shloka-bhalgat-unskript

Reply:
Changed @harshal-netserv